lib/screens/headways.ex

```diff
@@ -12,34 +12,36 @@ defmodule Screens.Headways do
   # Compact mapping of stop IDs to headway keys, leaning on the fact that subway stop IDs happen
   # to be numeric and often contiguous as we "traverse" the line in a given direction.
   @stops %{
-    blue_trunk: [70038..70060],
-    glx_medford: [70505..70514],
-    glx_union: [70503..70504],
+    blue_trunk: [70_038..70_060],
+    glx_medford: [70_505..70_514],
+    glx_union: [70_503..70_504],
```

Comment:
There is a lot of re-formatting noise here that doesn't seem necessary (the old version passed format checking as well).